pulseapi/profiles/views/profiles.py

```diff
@@ -242,7 +242,14 @@ class UserProfileListAPIView(ListAPIView):
 
     def get_queryset(self):
         request = self.request
-        queryset = UserProfile.objects.all().prefetch_related('related_user')
+        queries = self.request.query_params
```

Comment:
I noticed that we had a filter set up for the "is_active" flag, in an effort to keep this usable, I added this if statement.

That way, if a dev wants to visit the API and filter for all inactive users, they can still use the "?Is_active=False" search param.

Any other filtering however, will filter out inactive profiles by default.

Reply:
**Suggestion** It might make sense to add a test for this. Currently I believe this would return all profiles if `is_active` is part of the URL. E.g. `?is_active=True` and `?is_active=False` would return the same results. I think that's the case be cause the if-condition only checks if the parameter is in the included in the querystring, but not what its value is.

Reply:
Good Idea Tibor, I have added a new test where we:

- Create a test inactive profile to be sure there is one in the database.
- Preform a get request at the `profiles/?is_active` endpoint.
- Check if any of the returned profiles have `is_active` set to false.